examples/EventsExample.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+require __DIR__ . '/../vendor/autoload.php';
+
+use Monolog\Formatter\JsonFormatter;
+use Monolog\Handler\StreamHandler;
+use Monolog\Logger;
+use OpenTelemetry\Contrib\Zipkin\Exporter as ZipkinExporter;
+use OpenTelemetry\SDK\Common\Event\Dispatcher;
+use OpenTelemetry\SDK\Common\Event\Event\ErrorEvent;
+use OpenTelemetry\SDK\Common\Event\EventType;
+use OpenTelemetry\SDK\Common\Event\SimpleDispatcher;
+use OpenTelemetry\SDK\Common\Event\SimpleListenerProvider;
+use OpenTelemetry\SDK\Common\Log\LoggerHolder;
+use OpenTelemetry\SDK\Trace\SpanProcessor\SimpleSpanProcessor;
+use OpenTelemetry\SDK\Trace\TracerProvider;
+use Psr\Log\LogLevel;
+
+/**
+ * This example shows how you can hook in to OpenTelemetry's events, such as when an error or warning is emitted by
+ * the library. OpenTelemetry will always attempt to log and continue on error, and may provide no-op implementations
+ * of classes if it was unable to create them as requested.
+ * The default event handlers will use a PSR-3 logger to log events - verbosity and output format can be controlled
+ * through the usual PSR-3 log levels and formatters (if available).
+ * If you wish to do something different with events, you may provide an alternative PSR-14 implementation, but be
+ * aware that you will need to register handlers for all events that you are interested in.
+ */
+
+//logger used by default event handlers
+LoggerHolder::set(
```

Comment:
@tidal - this might be an opportunity to move `LoggerHolder` somewhere under `SDK\Event` ? A way to get rid of the static singleton could be to create something like an `AbstractLoggableEvent` (or trait) which accepts a `LoggerInterface` and is extended by our events as required?